Procfile

```diff
@@ -1,2 +1,2 @@
-release: ls network-api/app/networkapi && python network-api/app/manage.py migrate
+release: cd network-api && python network-api/app/manage.py --no-input migrate && python networkapi/app/manage.py heroku_release
```

Comment:
I believe this isn't gonna work. Double check it?

Reply:
yup, don't know what my brain was doing.